modules/media.js

```diff
@@ -354,34 +406,72 @@ Media.prototype.remove = function(filename, oncomplete) {
     throw new Error('Missing or invalid filename');
```

Comment:
Only that, in our use case, we'll only ever be removing files from an external SD card. That is, until support is added for other backup targets (usb, WiFi, etc).